asab/api/doc.py

```diff
@@ -31,14 +31,30 @@ def __init__(self, api_service, app, web_container, config_section_name="asab:do
 		self.WebContainer.WebApp.router.add_get("/asab/v1/openapi", self.openapi)
 
 		self.AuthorizationUrl = asab.Config.get(
-			config_section_name, "authorizationUrl", fallback=None
+			config_section_name, "authorization_url", fallback=None
 		)
-		self.TokenUrl = asab.Config.get(config_section_name, "tokenUrl", fallback=None)
+		self.TokenUrl = asab.Config.get(config_section_name, "token_url", fallback=None)
 		self.Scopes = asab.Config.get(config_section_name, "scopes", fallback=None)
 
+		# Deprecated options
+		# TODO: Remove them
+		if asab.Config.has_option(config_section_name, "authorizationUrl"):
```

Comment:
i don't think anyone is using these, so i suggest removing the deprecated options now